gradle.properties

```diff
@@ -9,16 +9,16 @@ ideVersionVerifier=IC-2023.1,IC-2024.3,IC-251.17181.16
 lombokVersion=1.18.32
 
 # alternative versions to simplify development and testing
-ideVersion=2023.1
```

Comment:
This must not be committed, because it defines the version to build a release build and we have to build against the earliest supported version.
Changing it locally only is fine, of course.

Reply:
I modified the run configuration instead, is that OK? It's just a bit annoying to have to modify the properties to run locally with the newer IDE. I always forget and have to go back, flip it, and restart. If I recently cleaned my caches it also adds waiting for the downloads, twice.